plugins/main/public/components/common/modules/overview-current-section.tsx

```diff
@@ -60,16 +65,7 @@ class WzCurrentOverviewSection extends Component {
   }
 
   render() {
-    return (
-      <span>
-        {/*this.props.currentTab && WAZUH_MODULES[this.props.currentTab] && WAZUH_MODULES[this.props.currentTab].title && (
-      <EuiTitle size='s'>
-        <h2>
-          {WAZUH_MODULES[this.props.currentTab].title}
-       </h2>
-      </EuiTitle>)*/}
-      </span>
-    );
```

Comment:
**thought**: is this being rendered before the change?

Reply:
I understand that this code is commented and that it is not rendered, or am I wrong?